app/controllers/api/service_dialogs_controller.rb

```diff
@@ -22,6 +22,7 @@ def refresh_dialog_fields_resource(type, id = nil, data = nil)
     def fetch_service_dialogs_content(resource)
       target, resource_action = validate_dialog_content_params(params)
       resource.content(target, resource_action, true)
+      normalize_value_ids(resource.content[0])
```

Comment:
Hmm, curious, if all we're doing is normalizing the id attribute in the response, why isn't the API normalizer doing it for us ?

Reply:
would you prefer i change the normalizer to also take symbol keys? this could be 'elsif attr.to_s == "id" || attr.to_s.ends_with?("_id")' which would solve this issue as well but it won't handle symbols at the moment

Reply:
it's just that this only is an issue for service dialogs and in particular only dialogs with fields that have `values` so potentially dropdowns and tag controls

Reply:
oh okay fine

Reply:
Thanks @d-m-u for the update.  This PR now fixes the underlying issue. If attr was to_s'd to check for _id names, it should have been so for checking id as well.  👍